### DISCUSSION TITLE
Is there a way to generalize the runner image to be used as a base image in vmss?

### DISCUSSION THREAD

**Discussion by xncbf** [community/community#160657] [Actions] [answered]
If there is any available documentation, please provide a link.

**Comment by code-on-git** [comment, marked as answer]
Maybe this is what you're looking for:
https://github.com/actions/runner-images/blob/main/docs/create-image-and-azure-resources.md

**Reply by xncbf** [reply]
This is what I was looking for. Thanks :)
